.eslintrc

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "@callstack",
+  "rules": {
+    "promise/always-return": "off"
```

Comment:
[nit] Why that rule?

Reply:
I had copied this config from the viem example repo. It's not needed here, I'll remove it